app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,66 @@
+// TODO
+// 1. Read Collection - GET /tasks
+// 2. Read Item - GET /tasks/{id}
+// 3. Create - POST /tasks
+// 4. Update - PUT/PATCH /tasks{id}
+// 5. Delete - DELETE /tasks/{id}
+
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.models.Tasks;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/tasks")
+@CrossOrigin
+public class TaskController {
+    private Tasks tasks = new Tasks();
+    private Long newId = 0L;
+
+    @GetMapping
+    public List<Task> list() {
+        return tasks.getTasks();
+    }
+
+    @GetMapping("/{id}")
+    public Task getTask(@PathVariable(name = "id") Long id) {
```

Comment:
PathVariable의 이름을 명시하는 것 같아요. 지금은 변수명과 완전히 같기 때문에 단순한게 생략해도 좋을 것 같아요.
```suggestion
    public Task getTask(@PathVariable Long id) {
```